### PR TITLE
Close stale issues automatically

### PR DESCRIPTION
**Pull request by casperklein**
20 days no activity --> mark as stale
another 10 days no activity --> close issue

For testing, I set for both 1 day. I will correct that, if it works as desired.

### CODE REVIEW

**Comment by aendeavor** [comment]
We (I) will need to place a new secret, the `secrets.GITHUB_TOKEN`. How do I go about this?

**Comment by casperklein** [comment]
No new secret is needed. See [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret)

**Review by aendeavor — APPROVED** [review]

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: 1 #20
```
Comment (radicand) [inline .github/workflows/stale.yml:16]: Is this set to 1 currently for testing purposes?

Comment (radicand) [reply .github/workflows/stale.yml:16]: Nevermind, I see the comment above.

Comment (casperklein) [reply .github/workflows/stale.yml:16]: Yes. See first comment ;-)

**Review by radicand — APPROVED** [review]

**Pull request merged**